tests/rptest/services/flink.py

```diff
@@ -193,6 +195,24 @@ def __init__(self, context, redpanda, topic, *args, **kwargs):
             self.STATE_FINISHED, self.STATE_CANCELED
         ]
 
+        _provider = context.globals['cloud_provider'].upper()
+        _meta = get_metadata_for_node(_provider, self.nodes[0])
+
+        # Handle service configuration
+        self.service_config = flink_config
+        # 1600m
+        self.service_config['jobmanager.memory.process.size'] = '27g'
+        # 1728m
```

Comment:
squash the commits into one? I think this commit is not really useful in the history.

Reply:
Agreed.

Reply:
Done